app/src/main/res/values/strings.xml

```diff
@@ -105,21 +105,20 @@
     <string name="affirm_accuracy_check_NotaBibleStory">(this story is not a Bible story)</string>
     <string name="request_accuracy_check_message">Please have someone with Bible knowledge check this story for accuracy.</string>
     <string name="stories_not_found_title">No stories found!</string>
-    <string name="stories_not_found_body">No story templates found! The currently selected folder
-        for story templates does not contain usable templates.  Select the appropriate
-        \'&lt;font color=\"green\">&lt;b>SP Templates&lt;/b>&lt;/font>\' folder - it may be on the SD card.
-    </string>
+    <string name="stories_not_found_body">It\'s time to install some story templates.&lt;br>
+        Tap the menu in the upper left, then...&lt;br>&lt;br>
+        To install the demo story, choose:&lt;br>
+        &lt;b>&lt;font color=\"white\">Add demo to story list&lt;/font>&lt;/b>&lt;br>&lt;br>
```

Comment:
Should "Add demo to story list" be title case here and on both the menus also should the other menu command be:
"Select SP Templates Folder"